lib/ex_unit/lib/ex_unit/doc_test.ex

```diff
@@ -466,7 +471,8 @@ defmodule ExUnit.DocTest do
           else: "#{indent} spaces"
 
         raise Error, line: line_no, module: module,
-                     message: "indentation level mismatch: #{inspect line}, should have been #{n_spaces}"
+                     message: "indentation level mismatch: #{inspect line}, " <>
```

Comment:
With addition of `<>` operation this whole construction starts to be overloaded with actions.
I think moving this to the:
```elixir
message =
  "indentation level mismatch: #{inspect line}, " <>
  "should have been #{n_spaces}"
```
will improve situation.